Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [OuterTune/App - OuterTune](https://hosted.weblate.org/projects/outertune/app/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/outertune/app/horizontal-auto.svg)

lucasmz-dev commented:
@DD3Boh is it OK if I ask Weblate to move the Libre "billing" over to your account? (You would need a Weblate account)
or @mikooomich ? I just wanna make sure it sticks with someone where it makes sense. It's on mine ATM because I created the project initially

mikooomich commented:
@lucasmz-dev Sure, just shoot me an email and we can discuss it further

lucasmz-dev commented:
I can create another PR with some smart squashing if you'd like (going to do it and just see)

mikooomich commented:
@lucasmz-dev I resolved the conflicts here. I don't believe there is a way to push to weblate, so I'm supposed to merge this into dev, then rebase/reset weblate, correct? https://github.com/OuterTune/OuterTune/commits/weblate-staging/

lucasmz-dev commented:
Yes exactly @mikooomich

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 96.2% (179 of 186 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/uk/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 96.7% (180 of 186 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/uk/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (186 of 186 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/uk/

Mickael81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (186 of 186 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/fr/

frezedev committed:
Translated using Weblate (Russian)

Currently translated at 97.3% (181 of 186 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ru/

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (186 of 186 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/pt_BR/

Commit:
Translated using Weblate (Russian)

Currently translated at 100.0% (186 of 186 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ru/

CheemsGalaxy committed:
Translated using Weblate (Vietnamese)

Currently translated at 44.0% (82 of 186 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/vi/

okdk66833 committed:
Translated using Weblate (Korean)

Currently translated at 100.0% (186 of 186 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ko/

suzuuuuu09 committed:
Translated using Weblate (Japanese)

Currently translated at 91.9% (171 of 186 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ja/

suzuuuuu09 committed:
Translated using Weblate (Japanese)

Currently translated at 93.5% (174 of 186 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ja/

CheemsGalaxy committed:
Translated using Weblate (Vietnamese)

Currently translated at 46.2% (86 of 186 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/vi/

Commit:
Translated using Weblate (Japanese)

Currently translated at 94.6% (176 of 186 strings)

Translation: OuterTune/OuterTune strings
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ja/
